### ISSUE TITLE
does mongrel work with daemons-1.1.0?

### ISSUE THREAD

**Issue by wenheping** [evan/mongrel#18]
Hi,

  Currently mongrel-1.2.0-pre2 require only daemons-1.0.10 but not

> = 1.0.10, since some distribution had update daemons to 1.1.0,
> it will leads mongrel-1.2.0-pre2 could not build.

  Would it be safe to change "daemons ~> 1.0.10" to " daemons >= 1.0.10" ?

 Thanks.

wen

Here is the patch:

--- tasks/gem.rake.orig 2011-02-11 16:32:01.000000000 +0000
+++ tasks/gem.rake      2011-02-11 16:32:16.000000000 +0000
@@ -12,7 +12,7 @@
   extra_rdoc_files << 'LICENSE'

   extra_deps << ['gem_plugin', '~> 0.2.3']
-  extra_deps << ['daemons', '~> 1.0.10']
-  extra_deps << ['daemons', '>= 1.0.10']
  
  extra_dev_deps << ['rake-compiler', "~> 0.7.0"]

**Comment by bibendi** [comment]
Yes, because of this dependence, we could not go to ruby 1.9.2 :(

**Comment by dcu** [comment]
any news?
